pyalgostrategypool/rsi_macd_crossover_v2.py

```diff
@@ -0,0 +1,76 @@
+import talib
+from pyalgotrading.constants import *
+from pyalgotrading.strategy import StrategyBase
+
+
+class RSIMACDCrossoverV2(StrategyBase):
```

Comment:
this class doesn't match with the one already in ready templates; that one's perfect, this has slight issues; fix it pls